lib/features/manage_account/data/repository/identity_repository_impl.dart

```diff
@@ -33,4 +35,14 @@ class IdentityRepositoryImpl extends IdentityRepository {
   Future<bool> editIdentity(AccountId accountId, EditIdentityRequest editIdentityRequest) {
     return _dataSource.editIdentity(accountId, editIdentityRequest);
   }
+  
+  @override
+  Future<List<Identity>?> getAllDefaultIdentityIds(AccountId accountId, {Properties? properties}) async {
```

Comment:
what is the different with `getAllIdentities`? IMO, we should find the smallest of identities here

Reply:
We have discuss it today, that default identity may be reused in multiple place, so that i think default identity is enough